sdk/cosmos/azure-cosmos/azure/cosmos/offer.py

```diff
@@ -24,13 +24,24 @@
 
 class ThroughputProperties(object):
     """Represents the throughput properties in an Azure Cosmos DB SQL API container.
-    To read and update throughput properties use the associated methods on the :class:`Container`.
+
+    To read and update throughput properties, use the associated methods on the :class:`Container`.
+    If configuring auto-scale, both `auto_scale_max_throughput` and `auto_scale_increment_percent` must be set and
```

Comment:
this is false, you only need auto_scale_max_throughput - please make sure all the documentation says as much

Reply:
will do